NewHorizons/Components/StarLightController.cs

```diff
@@ -96,6 +112,19 @@ public void Update()
                     break;
                 }
             }
+
+            if (PlayerState.InMapView())
+            {
+                foreach (var light in _lights) light.enabled = true;
+                return;
+            }
+
+            foreach (var light in _lights)
```

Comment:
This feels inefficient, we already loop through all stars now we're essentially looping through them all again to check their lights too. But I guess some stars don't have StarControllers so it isn't a 1:1 thing anyway.